_posts/2016-05-20-accountmanager.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Accountmanager New Business"
+date:   2016-05-07 14:22:53
+categories:
+permalink: accountmanager-new-business
+---
+
+Voor uitbreiding van ons succesvolle sales team zijn wij op zoek naar een Accountmanager New Business (m/v).
+
+## Functieprofiel
+
+De Accountmanager New Business die wij zoeken is in de eerste plaats geïnteresseerd in de software producten en de markten die Defacto bedient. Je bent verantwoordelijk voor het genereren van new business en denkt proactief mee over sales- en marketingactiviteiten. Doelmarkten; overheid, zorg en bedrijven met meer dan 1500 medewerkers.
```

Comment:
`in de eerste plaats` doesn't really make sense since there aren't any other interests required